BUILD.bazel

```diff
@@ -38,9 +38,14 @@ filegroup(
         "devtools/shared/**",
         "shells/**/*.js",
         "modalities/**/*",
+        "concrete-storage/*",
+        "concrete-storage/build/*",
+        "concrete-storage/platform/*",
+        "config/*",
     ]) + [
         "//src/tools:all_srcs",
         "//tools:tools_srcs",
+        ".eslintrc.json",
```

Comment:
Do you know why we needed to add this?

Reply:
Some version of the bazel tests needed it.